webapp/static/css/sticky-notes.css

```
@@ -72,3 +75,67 @@
+   שהיא מחליפה, כולל גלילה פנימית: פתק שגדל לבד מזיז את מה שסביבו. */
+.sticky-note-tasks {
+  width: 100%;
+  height: calc(100% - 42px);
+  overflow-y: auto;
+  padding: 8px;
+  font-size: 14px;
+  line-height: 1.35;
+  color: #222;
+  cursor: text;          /* הרמז שאפשר ללחוץ ולחזור לעריכה */
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)